packages/ui/src/scss/_desktop.scss

```diff
@@ -65,6 +65,7 @@ $font-family: -apple-system,
   --color-toast: #34353f;
   --color-overlay: rgba(0, 0, 0, 30%);
   // shadows
+  --shadow-1: 0 8px 16px rgba(66, 41, 159, 4%);
```

Comment:
`--shadow-1` is updated to `0 4px 8px rgba(0, 0, 0, 8%);`

Reply:
Sure, approved in prior. Update the shadow styles and it's good to merge

Reply:
checked updated.